### ISSUE TITLE
See playlist content?

### ISSUE THREAD

**Issue by Luna-Nocturna** [lmcintyre/OrchestrionPlugin#36]
Hallo,

First and foremost I want to thank for the work put into this plugin, I've used it for a long time now as I'd like some variety in my music during the grind this game can be sometimes. So many thanks for keeping this alive!

That said recently I've returned to the game after a little break and I noticed the favourites list has been replaced with playlists and we can even make multiple. This is amazing! However when I started to make some I couldn't figure out for my life how to access/view a playlist. I'm most likely missing something so the question is simply: How do I open the tracklist of a playlist so I can see what's in it?

Sorry if I'm asking a really obvious question because I miss out on some obvious button somewhere starting me right in my face.

**Comment by lmcintyre** [comment]
Sorry, I am not sure what you are asking. This is how you swap between viewing different playlists:

https://github.com/lmcintyre/OrchestrionPlugin/assets/6005409/babcbb37-8365-469d-9e6c-ceacd8793d71

**Comment by Luna-Nocturna** [comment]
Ok, so this was very interesting: Between the player and the playlists I just had a blank space, not a tracklist. I've wiped the plugin settings as a whole and re-installed it and it works as intended now. (Could it be because I updated from quite an older version which caused this issue?)

Either way thanks for your response. In hindsight I should've reset the plugin from the get go and not waste your time. My apologies.

Once again thank you for your work!
